projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -189,6 +215,35 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
     this.viewRef = null;
   }
 
+  /**
+   * This method is useful when you append to an element that you might remove from the DOM.
```

Comment:
For example, if you have a grid row with an element that you toggle using the display CSS property on hover.